csun-metalab/nomi
# NAM-404 -- UI: In course view, make previous, current, & next buttons equal width.

# Description
Made buttons equal width and added a check to see what course term the user is on for the button color.
  
# Testing
1. Make sure the button width is equal to each other.
2. Ensure the buttons are scale-able at different view-ports.
3. Make sure the correct color theme is applied to each button.
4. See if the color only applies to the correct term selected
